frontend/redux/nodes/components/ManageHostsPage/reducer.js

```diff
@@ -7,7 +7,7 @@ import {
 
 export const initialState = {
   page: 1,
-  perPage: 100,
+  perPage: 2,
```

Comment:
Change back to 100? Or remove this from state since it looks like it's in a constant now?

Reply:
ah yah my bad. was using that for testing

Reply:
removed